### ISSUE TITLE
Possible inaccurate profiling results for Liger Cross Entropy Kernel

### ISSUE THREAD

**Issue by yubofredwang** [Dao-AILab/quack#9]
Hi quack team,

Thanks for the awesome work! I have noticed that for Liger-Kernel Benchmarking on Cross Entropy forward kernel, the performance is significantly worse than both torch.compile and quack kernel. I wonder how the benchmarking is done? 

Asking because Liger's CE forward fused the gradient calculations as well, so basically its forward + backward combined. Want to doublecheck with the team on the benchmarking method as I noticed that CrossEntropy forward and backward are implemented separately in quack. Also, the benchmark script on the repo is also written for forward and backward separately.

Thanks!

**Comment by tridao** [comment]
I see, thanks so much for the clarification. We're redo the benchmark for liger and update the plot.
cc @GarlGuo @tzadouri: So we'd want to download liger and comment out the grad calculation in the code when benchmarking the forward.
